packages/rule-utils/src/colors.ts

```diff
@@ -58,9 +55,8 @@ export function colorToString(color: CSSColorValue | string, alphaOverride?: str
   alpha = alphaOverride ?? alpha
   type = type.toLowerCase()
 
-  // Comma separated functions
-  if (['hsla', 'hsl', 'rgba', 'rgb'].includes(type))
-    return `${type.replace('a', '')}a(${components.join(',')}${alpha == null ? '' : `,${alpha}`})`
+  if (['hsla', 'rgba'].includes(type))
+    return `${type}(${components.join(' ')}${alpha == null ? '' : ` / ${alpha}`})`
```

Comment:
Since we clearly agreed that we are going to align with Tailwind, I think this should use COMMA-SEPARATED syntax instead of SPACE-SEPARATED syntax.

Tailwind uses SPACE for `rgb` and `hsl`, but COMMA for `rgba` and `hsla`.

I can see that through out the MR we are changing everything to SPACE syntax, as well as `rgba` and `hsla`. This will work in the browser but that's not aligned with Tailwind.

Reply:
This will break for those who uses COMMA syntax with `rgba` and `hsla` (It makes sense to use legacy color function with legacy syntax? 🤣)

Reply:
I misunderstood it at first. I used tailwind play to check the output results and made adjustments.